python/lsst/ts/standardscripts/maintel/track_target_and_take_image_gencam.py

```diff
@@ -100,14 +117,34 @@ def get_schema(cls):
         schema_dict["title"] = "TrackTargetAndTakeImageGenCam v1"
         schema_dict["description"] = "Configuration for TrackTargetAndTakeImageGenCam."
 
-        schema_dict["camera_index"] = dict(
-            type="integer",
-            description="SAL Index for the Generic Camera.",
-            minimum=0,
-            maximum=2147483647,
+        # Leaving the option of ``camera_index`` being an integer for
```

Comment:
I think this comment is a bit misleading. You are removing ``camera_index`` no? I'd just remove it altogether.